user-guide/Advanced_Modules/SRM/Advanced_SRM_settings.md

```diff
@@ -36,7 +35,10 @@ namespace Script
         {
             var protocolFunctionHelper = new ProtocolFunctionHelper(engine.SendSLNetMessages);
             var currentConfig = protocolFunctionHelper.GetProtocolFunctionConfig();
-            currentConfig.ActiveFunctionResourcesThreshold = 123; // Change to desired value
+            // Change to the desired values
+            currentConfig.ActiveFunctionResourcesThreshold = 123;
+            currentConfig.FunctionHysteresis = TimeSpan.Zero;
```

Comment:
Not sure if we want to assign "TimeSpan.Zero" in the example, since this is not really a recommended value imo.

Reply:
I agree, but got it from the XML. I've changed that one too.